frontend/src/components/dialog/dialog.tsx

```diff
@@ -2,14 +2,24 @@ import * as DialogPrimitives from '@radix-ui/react-dialog'
 import * as React from 'react'
 import { animated, config, useTransition } from 'react-spring'
 
+import { Colors } from '../../config/colors'
+import { Title } from '../title'
+
 interface DialogProps {
   open: boolean
+  title?: string
   children: React.ReactNode
   size?: 'sm' | 'lg'
```

Comment:
i really don'y want to spend too much time on adjusting on the foundational components, since this is going to be moving in to the monorepo and will be using the ui-toolkit instead of duplicating everything.